### ISSUE TITLE
how to generate .dat file?

### ISSUE THREAD

**Issue by fengfengfen** [yaojieliu/ECCV2018-FaceDeSpoofing#3]
Hello Dr. Liu, in data_train.py, your code loads .dat file for training and testing. so how to generate these .dat files from original videos? Thank you!

**Comment by yaojieliu** [comment]
The .dat files are basically binary files without any header. For each sample, it consists of (256,256,3) uint8 image, (64,64,1) uint8 depth map, and 1 uint8 label. Tensorflow can use fixed length reader to read the data stream from the .dat files. 

This code is based tf1.4 and it is not the most effective way to prepare training data for tensorflow for now. I highly recommend to save the image and map label as png images, and use tf.data class to prepare the data. It's way more convenient to fixed length reader.  You just need to grab the network architecture and loss part.

If you want to use the whole code, please contact amin.jourabloo@gmail.com if you have any problem.

**Comment by Pro-flynn** [comment]
> Hello Dr. Liu, in data_train.py, your code loads .dat file for training and testing. so how to generate these .dat files from original videos? Thank you!
Hello，do you already know how to generate .dat file?
Could you give me some tips?
Thanks!

**Comment by Pro-flynn** [comment]
> Hello Dr. Liu, in data_train.py, your code loads .dat file for training and testing. so how to generate these .dat files from original videos? Thank you!

@fengfengfen
